src/base/intl_support.c

```diff
@@ -2726,6 +2726,36 @@ intl_strcasecmp_utf8_one_cp (const ALPHABET_DATA * alphabet, unsigned char *str1
   return 0;
 }
 
+/*
+ * intl_identifier_casecmp_for_dblinke() - compares two identifiers strings
+ *			       case insensitive excluding double quote for dblink
+ *
+ *   return: 0 if strings are equal, -1 if str1 < str2 , 1 if str1 > str2
+ *   str1(in):
+ *   str2(in):
+ *
+ * NOTE: this routine is the same as intl_identifier_casecmp
+ *       the first argument str1 may start with double quote but the second one never
+ */
+int
+intl_identifier_casecmp_for_dblink (const char *str1, const char *str2)
```

Comment:
Although explained in the NOTE, it is better to make a difference in the name of the argument argument.

What about dblink_col_name, remote_col_name?

Reply:
Ok. good point.